metrics/metrics.go

```diff
@@ -0,0 +1,31 @@
+package metrics
+
+import (
+	"log"
+	"runtime"
+	"time"
+
+	"gopkg.in/Clever/kayvee-go.v2"
+)
+
+func logMetric(source, metricName, metricType string, value uint64) {
+	payload := map[string]interface{}{"type": metricType, "value": value}
+	log.Printf(kayvee.FormatLog(source, kayvee.Info, metricName, payload))
+}
+
+// Log records Golang process metrics such as HeapAlloc, NumGC, etc... every
+// frequency time period. This function never returns so it should be called from
+// a Goroutine.
+func Log(source string, frequency time.Duration) {
+	for {
+		var memStats runtime.MemStats
```

Comment:
Can we move this after the goroutine log, so it's declared closer to where it's used?